src/pages/Home.tsx

```diff
@@ -1,3 +1,95 @@
-import { FC } from 'react';
+import { FC, useEffect, useState } from 'react';
 
-export const Home: FC = () => <h1>HOME</h1>;
+import { useUser } from '../hooks/useUser';
+import { PlantCard } from '../components/PlantCard';
+import { PlantType } from '../types/PlantType';
+import {
+	PlantsOrderBy,
+	fetchUserPlantsWithOrdering
+} from '../repository/fetchUserPlantsWithOrdering';
+
+type HomeTabs = 'water' | 'fertilize' | 'repot';
+
+export const Home: FC = () => {
+	const user = useUser();
+	const [tab, setTab] = useState<HomeTabs>('water');
+	const [userPlants, setUserPlants] = useState<PlantType[]>([]);
+
+	useEffect(() => {
+		let orderParameter: PlantsOrderBy = 'nextWater';
+		if (tab === 'fertilize') {
+			orderParameter = 'nextFertilize';
+		}
+		if (tab === 'repot') {
+			orderParameter = 'nextRepot';
+		}
+
+		const getUserPlants = async () => {
+			const plants = await fetchUserPlantsWithOrdering(
+				user?.email ?? '',
+				orderParameter
+			);
+
+			setUserPlants(plants);
+		};
+
+		getUserPlants();
+	}, [tab]);
+
+	return (
+		<div className="flex flex-col gap-4">
+			<div className="flex gap-4 items-center">
+				<h1 className="text-3xl">
+					Welcome{' '}
+					<span className="text-emerald-500 dark:text-emerald-700">
+						{user?.email}
+					</span>
+					!
+				</h1>
+			</div>
+
+			<h2 className="text-xl mt-4">Your plants status:</h2>
```

Comment:
I would maybe delete this heading, I think it would look better without it and it is not really neccessary as each of the tabs has a name (upcoming watering etc.), what do you think?